README.md

```diff
@@ -58,97 +61,84 @@ There are 1 API for prediction class
 ```ruby
 Datawow.prediction
 ```
----
-## Calling APIs
-Every classes there are 3 function that is `create`, `find_by` and `all`
-#### `create`
-```ruby
-Datawow.[class].create({data: "image URL", token: '_token'})
-```
-
-#### `find_by`
-```ruby
-Datawow.[class].find_by({id: "_image_id", token: '_token'})
-```
 
-#### `all`
+Those method, it's a shortcut for calling of these class
 ```ruby
-Datawow.[class].all({token: '_token'})
+Datawow::ImageClosedQuestion
+Datawow::PhotoTag
+Datawow::ImageChoice
+Datawow::ImageMessage
+Datawow::NanameueHuman
+Datawow::VideoClassification
+Datawow::TextClosedQuestion
+Datawow::TextCategory
+Datawow::TextConversation
+Datawow::TextJa
+Datawow::Prediction
 ```
 ---
-## Nanameue with Consensus
-##### There are 2 ways to use the library
-
-Firstly, __dynamic token__. if you have multiple project to work with, You could call library by using module name
+## Avalabile method for APIs
+Every classes there are 3 functions that is `create`, `find_by` and `all`
 #### `create`
 ```ruby
-Datawow::NanameueHuman.new('_token').create({data: "image URL"})
+Datawow.[class].create(data: {})
 ```
 
 #### `find_by`
 ```ruby
-Datawow::NanameueHuman.new('_token').find_by({id: "_image_id"})
+Datawow.[class].find_by({id: "_image_id"})
 ```
 
 #### `all`
 ```ruby
-Datawow::NanameueHuman.new('_token').all({page: '_page', per_page: '_per_page'})
+Datawow.[class].all()
 ```
 
-or another way
 
-```ruby
-model = Datawow::NanameueHuman.new
-model.project_key = '_token'
-model.create()
-```
+---
+## Example
 
-Secondly, we have initiated each modules since the library being called, so you could call from package name
+Since the library being called. we have initiated each modules, so you could call from package that we provided [here](#class_explanation)
```

Comment:
_(not sure if this is what you mean or not)_

After the library has been called for the first time, the associated modules will be initialised and you could then call using the package name instead. [Read more](#class_explanation)